src/main/java/com/github/dikhan/pagerduty/client/events/domain/Incident.java

```diff
@@ -32,6 +32,10 @@ protected Incident(IncidentBuilder builder) {
         this.links = builder.getLinks();
     }
 
+    public String toString() {
+        return getEventAction().getEventType() + " incident, routing_key: " + getRoutingKey();
```

Comment:
Can we add more info about the incident?

    @Override
    public String toString() {
        return "Incident{" +
                "routingKey='" + routingKey + '\'' +
                ", eventAction=" + eventAction +
                ", dedupKey='" + dedupKey + '\'' +
                ", payload=" + payload +
                ", images=" + images +
                ", links=" + links +
                '}';
    }